ocs_ci/helpers/dr_helpers.py

```
@@ -420,8 +421,20 @@ def check_mirroring_status_ok(
+            and "replicated-metadata-pool" not in cephbpradosns
+        ):
+            cephblockpool_rns_names = [
+                cephbprns_data["metadata"]["name"]
+                for cephbprns_data in ocp.OCP(
+                    kind=constants.CEPHBLOCKPOOLRADOSNS,
+                    namespace=config.ENV_DATA["cluster_namespace"],
+                ).get()["items"]
+            ]
+            cephbpradosns = list(
+                filter(lambda x: f"-{cephbpradosns}" in x, cephblockpool_rns_names)
+            )[0]
```

Ruff (0.15.12)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)